src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -151,12 +151,13 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
     // Initialisation
     if (node.layerUpdateState[layer.id] === undefined) {
         node.layerUpdateState[layer.id] = new LayerUpdateState();
+        const isInsideLimit = layer.tileInsideLimit(node, layer);
 
         if (!layer.tileInsideLimit(node, layer)) {
             // we also need to check that tile's parent doesn't have a texture for this layer,
             // because even if this tile is outside of the layer, it could inherit it's
             // parent texture
-            if (!layer.noTextureParentOutsideLimit &&
+            if (!isInsideLimit &&
```

Comment:
You're removing the code handling `layer.noTextureParentOutsideLimit` that you added a couple months ago.

Can you explain why it's not needed anymore?

(and technically this is an API breakage, so it would require deprecation handling)

Reply:
oops, it's a error